src/hotspot/share/jvmci/jvmciCompilerToVM.cpp

```diff
@@ -582,7 +582,7 @@ C2V_VMENTRY_NULL(jobject, lookupType, (JNIEnv* env, jobject, jstring jname, ARGU
   TempNewSymbol class_name = SymbolTable::new_symbol(str);
 
   if (class_name->utf8_length() <= 1) {
-    JVMCI_THROW_MSG_0(InternalError, err_msg("Primitive type %s should be handled in Java code", class_name->as_C_string()));
+    JVMCI_THROW_MSG_0(InternalError, err_msg("Primitive type %s should be handled in Java code", str));
```

Comment:
Seems unrelated to the fix at hand.

Reply:
Yes, it's a minor fix up I noticed while making changes a few lines below. It just avoids a conversion of a `Symbol` back to a C string when the original C string is available.